presets/4.4/tune/uav_tech/UAV_tech_Whoop.txt

```diff
@@ -50,25 +56,80 @@ set simplified_pitch_d_gain = 100
 set simplified_pitch_pi_gain = 100
 set simplified_master_multiplier = 160
 
-simplified_tuning apply
+set iterm_relax_cutoff = 5
+set vbat_sag_compensation = 100
+set anti_gravity_gain = 90
+set pidsum_limit = 1000
+set pidsum_limit_yaw = 1000
+
+#$ OPTION_GROUP BEGIN: Choose ONE Filter option (+ RPM filter if desired)
+    #$ OPTION BEGIN (UNCHECKED): low Build Quality
+    	# -- ADDER: For HIGH gyro vibration builds --
+    	set simplified_gyro_filter = ON
+    	set simplified_gyro_filter_multiplier = 40
+    	set simplified_dterm_filter = ON
+    	set simplified_dterm_filter_multiplier = 100
+    	set dyn_notch_count = 4
+    	set dyn_notch_min_hz = 80
+    	set dyn_notch_max_hz = 700
+    	set yaw_lowpass_hz = 90
+    #$ OPTION END
 
-#$ OPTION BEGIN (UNCHECKED): 48k ESC PWM
-    # -- ADDER: For 48khz PWM --
-    set thrust_linear = 20
-#$ OPTION END
+    #$ OPTION BEGIN (CHECKED): Medium Build Quality
+    	# -- ADDER: For Medium gyro vibration builds --
+    	set simplified_gyro_filter = ON
+    	set simplified_gyro_filter_multiplier = 60
+    	set simplified_dterm_filter = ON
+    	set simplified_dterm_filter_multiplier = 120
+    	set dyn_notch_count = 3
+    	set dyn_notch_min_hz = 100
+    	set dyn_notch_max_hz = 700
+    	set yaw_lowpass_hz = 125
+    #$ OPTION END
 
-#$ OPTION BEGIN (UNCHECKED): 96k ESC PWM
-    # -- ADDER: For 96khz PWM --
-    set thrust_linear = 40
-#$ OPTION END
+    #$ OPTION BEGIN (UNCHECKED): HIGH Build Quality
+    	# -- ADDER: For low gyro vibration builds --
+    	set simplified_gyro_filter = ON
+    	set simplified_gyro_filter_multiplier = 120
+    	set simplified_dterm_filter = ON
+    	set simplified_dterm_filter_multiplier = 120
+    	set dyn_notch_count = 2
+    	set dyn_notch_min_hz = 125
+    	set dyn_notch_max_hz = 700
+    	set yaw_lowpass_hz = 0
+    #$ OPTION END
 
-#$ OPTION BEGIN (UNCHECKED): RPM filtering
-    set dshot_bidir = ON
-    set rpm_filter_harmonics = 2
-    set dyn_notch_count = 2
-#$ OPTION END
+    #$ OPTION BEGIN (UNCHECKED): ... + RPM filter (ESC bi-directional Dshot required)
+    	# -- ADDER: Enabled RPM filtering --
+    	set motor_pwm_protocol = DSHOT300
+    	set dshot_bidir = ON
```

Comment:
since this is whoop, i guess the poles count is known? Can set it up here for better user experience.